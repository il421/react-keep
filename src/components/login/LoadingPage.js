import React from 'react';
import { RingLoader } from 'react-spinners';

const LoadingPage = () => (
  <div className="loader">
    <RingLoader
      sizeUnit={'px'}
      size={ 100 }
      color={'#4abdac'}
      loading={ true }
    />
  </div>
);

export default LoadingPage;
