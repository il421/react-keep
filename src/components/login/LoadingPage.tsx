import * as React from "react";
import { RingLoader } from "react-spinners";
import { AlignItems, Colors, JustifyContent } from "../../common";
import "../../styles/components/login/_loading-page.scss";
import { FlexBox } from "../ui-components";

const LoadingPage = (): JSX.Element => (
  <FlexBox
    vertical={true}
    alignItems={AlignItems.center}
    justifyContent={JustifyContent.center}
    className="loading-page"
  >
    <RingLoader size={100} color={Colors.fresh} loading={true} />
    <div>We are getting your important notes...</div>
  </FlexBox>
);

export default LoadingPage;
