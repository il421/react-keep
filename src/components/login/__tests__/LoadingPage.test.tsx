import { shallow, ShallowWrapper } from "enzyme";
import * as React from "react";

import LoadingPage from "../LoadingPage";

test("should render LoadingPage correctly", () => {
  const wrapper: ShallowWrapper<typeof LoadingPage> = shallow<
    typeof LoadingPage
  >(<LoadingPage />);
  expect(wrapper.debug()).toMatchSnapshot();
});
