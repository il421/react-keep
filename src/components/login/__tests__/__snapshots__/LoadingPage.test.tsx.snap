// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render LoadingPage correctly 1`] = `
"<FlexBox vertical={true} alignItems=\\"center\\" justifyContent=\\"center\\" className=\\"loading-page\\">
  <Loader size={100} color=\\"#8ac0b8\\" loading={true} css=\\"\\" />
  <div>
    We are getting your important notes...
  </div>
</FlexBox>"
`;
